shared/src/main/scala/minitest/api/SourceLocation.scala

```diff
@@ -34,7 +34,7 @@ object SourceLocation {
     val pos = tasty.rootPosition
     val fileName = pos.sourceFile.getFileName.toString
     val path = pos.sourceFile.getParent.toString
-    val line = pos.startLine
+    val line = pos.startLine + 1
```

Comment:
Why is `+ 1` needed? is `pos.startLine` broken?

Reply:
No, `startLine` is the index of the start line but the IDEs start with line 1.